### PR TITLE
Set SpanKind for HTTP and gRPC requests

### PR DESCRIPTION
**Pull request by rakyll**

### CODE REVIEW

File: plugin/ocgrpc/client.go [hunk 1]
```diff
@@ -25,6 +25,8 @@ import (
 // traces. Use with gRPC clients only.
 type ClientHandler struct {
 	// StartOptions allows configuring the StartOptions used to create new spans.
+	//
+	// SpanKind will always set to SpanKindClient for spans started with this handler.
```
Comment (odeke-em) [inline plugin/ocgrpc/client.go]: `will always be set to SpanKindClient for spans started by`

File: plugin/ocgrpc/server.go [hunk 2]
```diff
@@ -46,6 +46,8 @@ type ServerHandler struct {
 	// ensures that each service has some opportunity to be traced. If you would
 	// like to not add any additional traces for this gRPC service, set:
 	//   StartOptions.Sampler = trace.ProbabilitySampler(0.0)
+	//
+	// SpanKind will always set to SpanKindServer for spans started from this handler.
```
Comment (odeke-em) [inline plugin/ocgrpc/server.go]: ditto

File: plugin/ocgrpc/client.go (hunk 1 above)
Comment (rakyll) [reply plugin/ocgrpc/client.go]: Done

File: plugin/ocgrpc/server.go (hunk 2 above)
Comment (rakyll) [reply plugin/ocgrpc/server.go]: Done

**Pull request merged**